src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/KnownCoseAlgorithms.cs

```diff
@@ -16,6 +16,10 @@ internal static class KnownCoseAlgorithms
         internal const int PS256 = -37;
         internal const int PS384 = -38;
         internal const int PS512 = -39;
+        // RSASSA-PKCS1-v1_5 using SHA
+        internal const int RS256 = -257;
+        internal const int RS384 = -258;
+        internal const int RS512 = -259;
```

Comment:
`FromString` and `ThrowIfNotSupported` don't seem to account for the new RSXYZ constants. Should they?

Reply:
I covered the gaps of the new Pkcs1 algorithms on https://github.com/dotnet/runtime/pull/71390/commits/5fb06cc159d0d65c89fd177708830180462101c2.